conda/switch_cuda_version.sh

```diff
@@ -17,11 +17,18 @@ mkdir -p /usr/local
 rm -fr /usr/local/cuda
 ln -s "$CUDA_DIR" /usr/local/cuda
 
+CUDA_VERSION_DOT=$(nvcc --version | sed -n 4p|cut -f5 -d" " | cut -f1 -d",")
```

Comment:
maybe copy the comment from PR on why using `nvcc --version` --> since nvcc is part of dev not runtime. good to explain that it requires the whole package. not just cuda runtime.

Reply:
also question - did we try using nvidia-smi?

Reply:
no I haven't used nvidia-smi